tests/Console/Command/DescribeCommandTest.php

```diff
@@ -32,18 +34,31 @@
  */
 final class DescribeCommandTest extends TestCase
 {
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
```

Comment:
>By side effect I mean: the instance is used in some way in a test that may affect another test as they share the same instance.

please provide a POC for this

>Not sure it's PHPUnit's responsability to do that, could make sense. But I'm pretty sure it doesn't currently. Why setting the property to null wouldn't help?

it is, we never set a test class to null

>Why setting the property to null wouldn't help?

why would it?